glx/glxdri2.c

```diff
@@ -448,7 +448,7 @@ create_driver_context(__GLXDRIcontext * context,
     context->driContext = NULL;
 
     if (screen->dri2->base.version >= 3) {
-        uint32_t ctx_attribs[4 * 2];
+        uint32_t ctx_attribs[4 * 2] = { NULL };
```

Comment:
`uint32_t` isn't a pointer,
```suggestion
        uint32_t ctx_attribs[4 * 2] = { 0 };
```

Reply:
Many thanks for fix my typo and init array warnings.